routes/api/user/delete-user.js

```diff
@@ -7,5 +7,34 @@ var router = express.Router();
 
 console.log('delete-user');
 
+app.get("/api/users", function(req, res) {
+    // 1. Add a join to include all ofusers
+    db.User.findAll({}).then(function(dbUser) {
+      res.json(dbUser);
+    });
+  });
+
+  app.get("/api/users/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.User.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbUser) {
+      res.json(dbUser);
+    });
+  });
```

Comment:
These aren't necessary here and can be removed. They're already in the `read-user.js` file